packages/amplify-codegen/package.json

```diff
@@ -37,7 +37,8 @@
     "js-yaml": "^4.0.0",
     "ora": "^4.0.3",
     "semver": "^7.3.5",
-    "slash": "^3.0.0"
+    "slash": "^3.0.0",
+    "amplify-prompts": "^1.6.2"
```

Comment:
Shouldn't this be in peer dependencies?

Reply:
Our code takes a direct dependency on it.